File: DotNet/SETPractice/SOLIDPrinciples/DependancyInversion/EmployeeDataAccessLayer.cs
namespace SOLIDPrinciples.DependancyInversion
{
    public class EmployeeDataAccessLayer : IEmployeeDataAccessLayer
    {
        public Employee GetEmployee(int id)
        {
            return new Employee()
            {
                EmpId = id,
                Name = "Sudhan",
                Department = "IT",
                Salary = 100000
            };
        }
    }
}

File: Java/SETPractice/src/main/java/SOLIDPrinciple/DependencyInversion/EmployeeDataAccessLayer.java
package SOLIDPrinciple.DependencyInversion;

public class EmployeeDataAccessLayer implements IEmployeeDataAccessLayer{
    @Override
    public Employee GetEmployee(int id) {
        Employee emp = new Employee();
        emp.EmpId = id;
        emp.Name = "Sudhan";
        emp.Department = "IT";
        emp.Salary = 100000;
        return emp;
    }
}
